include/overlay021/struct_ov21_021D0D80.h

```diff
@@ -9,11 +9,11 @@
 typedef struct {
```

Comment:
**polish**: Do both this struct and `ov21_021D0D80_1` below need to exist?

Can this struct be merged into a parent header?